src/components/Listbox.vue

```diff
@@ -117,8 +127,9 @@ export default defineComponent({
 })
 </script>
 <template>
-  <Listbox v-model="model" as="div" class="group relative">
+  <Listbox v-model="model" as="div" class="group" :class="{ 'relative': !escapeOverflow }">
```

Comment:
by removing `relative`, the absolutely-positioned options panel does not position itself anymore relatively to the button. But that makes it lose the width constraints, which I had to re-add via JavaScript :(